app.rb

```diff
@@ -0,0 +1,143 @@
+require './book'
+require './student'
+require './teacher'
+require './person'
+require './rental'
+
+class App
+  include Inputs
+  def initialize
+    @people = []
+    @books = []
+    @id = 0
+  end
+
+  attr_accessor :persons, :books
+
+  def run
+    puts 'Welcome to School Library App!'
+    display_options
+
+    task_list = {
+      '1' => method(:list_all_books),
+      '2' => method(:list_all_people),
+      '3' => method(:create_person),
+      '4' => method(:create_book),
+      '5' => method(:create_rental),
+      '6' => method(:list_rentals)
```

Comment:
- [ ] There is no ruby inbuilt method called `method`. I also checked all the ruby files in your directory and there is no custom function called `method`. Kindly recheck the method you are calling here. 👍